scenes/IngameScores.gd

```diff
@@ -0,0 +1,31 @@
+extends VBoxContainer
+
+const score_viewer := preload("res://scenes/performance/ingame_user_score_viewer.tscn")
+
+func reload_users(ingame_users : Dictionary) -> void:
+	for c in get_children():
+		c.name = "__tmp"
+		c.queue_free()
+	load_users(ingame_users)
+
+func load_users(ingame_users : Dictionary) -> void:
+	for k in ingame_users:
+		var n = get_node_or_null(k)
+		if !n:
```

Comment:
GDScript style prefers words over symbols in boolean statements, so "if not n:". Only use exclamation mark for unequal symbol, !=